scripts/Glacier2Util.py

```diff
@@ -5,7 +5,7 @@
 import sys, os
 from Util import *
 
-class Glacier2Router(ProcessFromBinDir, ProcessIsReleaseOnly, Server):
+class Glacier2Router(ProcessIsFromPath, ProcessFromBinDir, ProcessIsReleaseOnly, Server):
```

Comment:
Why do we keep ProcessFromBinDir here, is this temporary?

Reply:
We could probably remove it in a cleanup pass later.  I can remove it now if you rather?

Reply:
If it is no longer require just remove it.

Reply:
ok, I removed it from the glacier2 and icegrid classes